packages/shared/src/components/cards/v1/SharedPostCardFooter.tsx

```diff
@@ -6,44 +6,35 @@ import { IconSize } from '../../Icon';
 import { CardImage, CardVideoImage } from './Card';
 
 type SharedPostCardFooterProps = {
-  isShort: boolean;
   isVideoType?: boolean;
 } & Pick<Post, 'sharedPost'>;
 
 export const SharedPostCardFooter = ({
   sharedPost,
-  isShort,
   isVideoType,
 }: SharedPostCardFooterProps): ReactElement => {
   const ImageComponent = isVideoType ? CardVideoImage : CardImage;
   return (
     <div
       className={classNames(
-        'mb-2 flex h-auto min-h-0 w-full flex-auto gap-3 rounded-12 border border-theme-divider-tertiary p-3',
-        isShort ? 'flex-row items-center' : 'flex-col  mobileL:flex-row',
+        'mb-2 flex h-auto min-h-0 w-full flex-auto flex-col gap-3 rounded-12 border border-theme-divider-tertiary p-3  mobileL:flex-row',
       )}
     >
-      <div
-        className={classNames(
-          'flex flex-col mobileL:flex-1',
-          isShort ? 'line-clamp-4 w-8/12 pr-3' : 'line-clamp-2',
-        )}
-      >
-        {sharedPost.title}
-        <span className="text-theme-label-quaternary">
+      <div className={classNames('line-clamp-1 flex flex-col mobileL:flex-1')}>
```

Comment:
Why would the wrapping div have a `line-clamp-1` if the children have different line clamps?

Reply:
this class applied to the div is actually completely ignored -- removed 🙏